tests/contrib/operators/test_dataproc_operator.py

```diff
@@ -270,6 +278,27 @@ def test_cluster_name_log_sub(self):
                 mock_info.assert_called_with('Creating cluster: %s',
                                              u'smoke-cluster-testnodash')
 
+    def test_build_cluster_data_internal_ip_only_without_subnetwork(self):
+
+        def create_cluster_with_invaild_internal_ip_only_setup():
+
+            # Guiven
+            create_cluster = DataprocClusterCreateOperator(
+                task_id=TASK_ID,
+                cluster_name=CLUSTER_NAME,
+                project_id=PROJECT_ID,
+                num_workers=NUM_WORKERS,
+                zone=ZONE,
+                dag=self.dag,
+                internal_ip_only=True)
+
+            # When
+            create_cluster._build_cluster_data()
+
+        # Then
+        self.assertRaises(AirflowException,
+                          create_cluster_with_invaild_internal_ip_only_setup)
+
```

Comment:
Can you also just add another line to actually check if the Exception message contains "Set internal_ip_only to true only when you pass a subnetwork_uri."?

Reply:
Once that is done I think it can be merged.